### DISCUSSION TITLE
New to Homebrew

### DISCUSSION THREAD

**Discussion by arya2331** [Homebrew/discussions#88] [Everyday usage]
Hello, I am new to homebrew and wanted to contribute to it. Can someone guide me on how to start contributing to it? I know little Ruby and I am willing to learn it.

**Comment by SMillerDev** [comment]
First, find something that you want changed. Then change it. If it's formulae you want to change there's some guides here: https://docs.brew.sh/

**Reply by arya2331** [reply]
How can I find something that needs to be changed? I am completing new to this. Can you suggest some task or issues to get started with?

**Reply by SMillerDev** [reply]
Pick something you think Homebrew could do better. We don't really have ready made tasks.

**Reply by jonchang** [reply]
https://github.com/search?q=org%3AHomebrew+label%3A%22help+wanted%22+is%3Aopen has a list of open issues and pull requests labeled "help wanted".
